.github/workflows/release.yaml

```diff
@@ -121,6 +121,58 @@ jobs:
             - name: Publish to NPM
               run: npm publish --tag latest
 
+    version-docs:
+        needs: publish_to_npm
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v5
+              with:
+                  ref: ${{ github.event.repository.default_branch }}
+                  token: ${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}
+                  fetch-depth: 0
+
+            - name: Use Node.js 22
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 22
+
+            - name: Install jq
+              run: sudo apt-get install jq
+
+            - name: Activate cache for Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                  cache: 'yarn'
+
+            - name: Install dependencies
+              run: |
+                  # install project deps
+                  yarn
+                  # install website deps
+                  cd website
+                  yarn
+
+            - name: Snapshot the current version
+              run: |
+                  cd website
+                  VERSION=$(jq -r '.version' ../package.json)
+                  MAJOR=$(echo $VERSION | cut -d. -f1)
+                  rm -rf versioned_docs/version-$MAJOR
+                  rm -rf versioned_sidebars/version-$MAJOR-sidebars.json
+                  jq --arg major "$MAJOR" 'map(select(. != $major))' version.json > tmp.json && mv tmp.json version.json
```

Comment:
This magic incantation.... I love it, and I'm sure it works perfectly fine, but wow is this ever smth that we could've probably done in a nodejs script instead 😄